Package.swift

```diff
@@ -18,7 +22,7 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(url: "https://github.com/mapbox/mapbox-common-ios.git", .exact(mapboxCommonVersion)),
+        .package(url: "https://github.com/mapbox/mapbox-common-ios.git", versionRangeThatAllowsOnlyPatches(for: mapboxCommonVersion)),
```

Comment:
This is equivalent to `Package.Dependency.Requirement.upToNextMinor(from:)`, which would be clearer than the custom function used here. However, CocoaPods and Carthage don’t quite support the same semantics, at least not for post-v1.0 packages. The only common denominator that works regardless of package manager is `.upToNextMajor(from:)`, which is incidentally the [officially recommended](https://developer.apple.com/documentation/swift_packages/package/dependency/3582850-package#discussion) practice on iOS.